exporter/datadogexporter/internal/metadata/metadata.go

```diff
@@ -59,6 +60,13 @@ type HostMetadata struct {
 
 	// Tags includes the host tags
 	Tags *HostTags `json:"host-tags"`
+
+	//Gohai payload contains inventory of system information
```

Comment:
```suggestion
	// Payload contains inventory of system information.
```
This is not very clear. Can you be more specific. What does it actually contain?